s4sim/hardware/config.py

```diff
@@ -1641,67 +1651,46 @@ def get_example():
         "LT253",
         "LT254"
     ]
-    tele["platescale"] = 0.0047619
-    # This tube spacing in mm corresponds to 0.83 degrees projected on
-    # the sky at a plate scale of 210 mm/deg or 0.0047619 deg/mm
-    # The physical tube spacing is actually 210 mm.
-    tele["tubespace"] =  174.3
+    tele["platescale"] = 0.00429
+    # This tube spacing in mm corresponds to 0.94 degrees projected on
+    # the sky at a plate scale of 0.00429 deg/mm
+    # The physical tube spacing may be different.
+    tele["tubespace"] =  0.94/0.00429
     tele["fwhm"] = lfwhm
     telescopes["LAT2"] = tele
 
     #from the DSR
     sfwhm = OrderedDict()
-    sfwhm["SAT_f030"] = 72.8
-    sfwhm["SAT_f040"] = 72.8
-    sfwhm["SAT_f085"] = 25.5
-    sfwhm["SAT_f145"] = 25.5
-    sfwhm["SAT_f095"] = 22.7
-    sfwhm["SAT_f155"] = 22.7
-    sfwhm["SAT_f220"] = 13.0
-    sfwhm["SAT_f280"] = 13.0
+    sfwhm["SAT_f030"] = 85.2
```

Comment:
Some significant changes here... We will have to double check that @zonca  has used the appropriate FWHM to generate the beam-smoothed skies.

Reply:
I used https://github.com/CMB-S4/s4mapbasedsims/blob/master/202102_design_tool_run/instrument_model/cmbs4_instrument_model.tbl

Reply:
@zonca I'm afraid the numbers in your table match the old ones. We will need to update the SPLAT and SPSAT simulations.

Reply:
Yes, these changed signifcantly